far/keyboard.cpp

```diff
@@ -1730,10 +1730,18 @@ int TranslateKeyToVK(int Key, INPUT_RECORD* Rec)
 						KEY_DOWN,
 						KEY_INS,
 						KEY_DEL,
-						KEY_NUMENTER
+						KEY_NUMENTER,
+						//todo Browser*, Launch*, Media*, ?Standby, ?Spec*, ?Oem*
+						KEY_LWIN,
+						KEY_RWIN,
+						KEY_APPS,
+						KEY_PRNTSCRN,
+						KEY_BREAK,
+						KEY_DIVIDE,
+						KEY_NUMLOCK
 					};
 
-					if (contains(ExtKey, FKey))
+					if (contains(ExtKey, FKey) || VirtKey==VK_RCONTROL || VirtKey==VK_RMENU)
```

Comment:
VirtKey deduction can be ambiguous:
https://github.com/FarGroup/FarManager/blob/13b99b1d19a279de6bce350e0c8d945677f5323c/far/keyboard.cpp#L1605-L1618

Maybe check for `Key == KEY_RCTRL || Key == KEY_RALT`?

Reply:
> Maybe check for Key == KEY_RCTRL || Key == KEY_RALT?

Hm, that actually was meant to also work with combinations like <kbd>Ctrl</kbd>+<kbd>RAlt</kbd>
But I see now that using VirtKey does not help as well...

Reply:
If you want to support CtrlAlt etc. the last one should be treated as a key (`FKey` above) and the rest as modifiers (`FShift` above), which is currently not the case.

Reply:
Fixed for now.